### PR TITLE
fix: add doc for #1358

### PR DESCRIPTION
**Pull request by lut777**
a doc to describe solutions for #1358

### CODE REVIEW

File: dist/images/kubectl-ko [hunk 1]
```diff
@@ -375,7 +375,7 @@ getOvnCentralPod(){
     if [ -z "$VERSION" ]; then
           echo "kubeovn version not exists"
           exit 1
-        fi
+    fi
```
Comment (oilbeater) [inline dist/images/kubectl-ko:378]: this file doesn't belong to this pr

File: dist/images/dbcheck_tmp.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (lut777) [reply dist/images/dbcheck_tmp.sh]: deleted

File: dist/images/kubectl-ko (hunk 1 above)
Comment (lut777) [reply dist/images/kubectl-ko]: deleted

**Review by oilbeater — APPROVED** [review]
